pokepaylib/build.gradle

```diff
@@ -50,13 +48,63 @@ dependencies {
 
 def repo = new File(rootDir, "repository")
 apply plugin: 'maven'
+task androidSourcesJar(type: Jar) {
+    archiveClassifier.set('sources')
+    from android.sourceSets.main.java.srcDirs
+}
+artifacts {
+    archives androidSourcesJar
+}
+
 uploadArchives {
     repositories {
         mavenDeployer {
             repository url: "file://${repo.absolutePath}"
             pom.version = '1.6.1'	      // version
+            beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
+
+            repository(url: "https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/") {
+                authentication(userName: ossrhUsername, password: ossrhPassword)
+            }
+
+            snapshotRepository(url: "https://s01.oss.sonatype.org/content/repositories/snapshots/") {
+                authentication(userName: ossrhUsername, password: ossrhPassword)
+            }
+
+            pom.project {
+                name 'Pokepay Android SDK'
+                packaging 'aar'
+                // optionally artifactId can be defined here
+                description 'This is an SDK for embedding Pokepay\'s payment platform into Android applications.'
+                url 'https://docs.pokepay.jp/guidelines/app-sdk/java.html'
+
+                scm {
+                    connection 'scm:git:git://github.com/pokepay/android-sdk.git'
+                    developerConnection 'scm:git:ssh://github.com/pokepay/android-sdk.git'
+                    url 'https://github.com/pokepay/android-sdk'
+                }
+
+                licenses {
+                    license {
+                        name 'The Apache License, Version 2.0'
+                        url 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                    }
+                }
+
+                developers {
+                    developer {
+                        id 'ann.lin'
+                        name 'Ann Lin'
+                        email 'ann.lin@coilinc.jp'
+                    }
```

Comment:
@WeiWW Is this part necessary to add other developers like tateishi-san?

Reply:
Just temporary, I'll survey and change it later, then the dev account can do this.

Reply:
ok